### ISSUE TITLE
Some Keys that use AltGr in fr-FR keyboards do not work in 2.0.0

### ISSUE THREAD

**Issue by fMichaleczek** [PowerShell/PSReadLine#832]
With PSReadLine 2.0.0-lastCI on French Keyboard, press AltGr key and some characters ( **_[ ] \ ^_** )  display an additional character **_^_** 

Environment data
----------------

```powershell
PS version: 5.1.17134.407
PSReadline version: 2.0.0-beta3
os: 10.0.17134.1 (WinBuild.160101.0800)
PS file version: 10.0.17134.1 (WinBuild.160101.0800)
```

Steps to reproduce or exception report
--------------------------------------

```powershell
On a French Keyboard, press : 
Alt Gr + 5 
Alt Gr + 8
Alt Gr + 9
Alt Gr + °
```
Expected behavior

```none
[
\
^
]
```

Actual behavior

```none
^[
^\
^^
^]
```

**Comment by lzybkr** [comment]
Assuming this is related to https://github.com/lzybkr/PSReadLine/pull/831 - maybe you should comment there instead and close this.

**Comment by sba923** [comment]
So? Where do you want us to log the issues with the French keyboard?

I'm running beta4 in PowerShell 6.2.0 and am experiencing the following:

- in one of my PS6.2 hosts I get the following:
* AltGr+2 (dead key) followed by SPACE yields `@~` instead of `~` (the `@` is displayed immediately, before I hit SPACE)
* AltGr+7 (dead key) followed by SPACE yields ``@` `` instead of `` ` `` (the `@` is displayed immediately, before I hit SPACE)
- in another PS6.2 host I get the following:
* AltGr+2 (dead key) followed by SPACE yields `é~` instead of `~` (the `é` is displayed immediately, before I hit SPACE)
* AltGr+7 (dead key) followed by SPACE yields ``é` `` instead of `` ` `` (the `è` is displayed immediately, before I hit SPACE)

Workaround: Use Ctrl+Alt+2 / Ctrl+Alt+7 -- but doesn't always work (did when I originally posted, doesn't right now).

**Comment by daxian-dbw** [comment]
I believe this has been fixed by #933 and #945 (related issue: #914).
Please try the `2.0.0-beta5` PSReadLine from PowerShellGallery.
